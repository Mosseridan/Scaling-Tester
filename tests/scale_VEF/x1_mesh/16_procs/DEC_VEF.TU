Statistiques de post resolution

Temps total                       0.659745

