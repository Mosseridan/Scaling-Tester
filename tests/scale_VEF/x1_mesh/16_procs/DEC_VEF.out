-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VEF
 
Reading data file 

14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
14 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 16
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[6.25e-02]   1=[6.25e-02]   2=[6.25e-02]   3=[6.25e-02]   4=[6.25e-02]
        5=[6.25e-02]   6=[6.25e-02]   7=[6.25e-02]   8=[6.25e-02]   9=[6.25e-02]
       10=[6.25e-02]  11=[6.25e-02]  12=[6.25e-02]  13=[6.25e-02]  14=[6.25e-02]
       15=[6.25e-02]
   Allowed maximum load imbalance: 1.001 

      3672      13800      13800 [20] [      275:    3672 ]
      1836       7636      10128 [20] [      275:    3672 ]
       918       4368       6456 [20] [      275:    3672 ]
       497       2832       4114 [20] [      275:    3672 ]
       267       1672       3180 [20] [      275:    3672 ]
       145       1022       2482 [20] [      275:    3672 ]
        76        596       1932 [20] [      275:    3672 ]
        40        298       1574 [20] [      275:    3672 ]
        21        136       1286 [20] [      275:    3672 ]
        19        116       1244 [20] [      275:    3672 ]
      1836       6826       6826 [20] [      137:    1836 ]
       918       3704       4990 [20] [      137:    1836 ]
       459       2134       3154 [20] [      137:    1836 ]
       250       1382       1992 [20] [      137:    1836 ]
       136        796       1548 [20] [      137:    1836 ]
        74        484       1180 [20] [      137:    1836 ]
        40        286        942 [20] [      137:    1836 ]
        22        140        740 [20] [      137:    1836 ]
        18        116        718 [20] [      137:    1836 ]
       916       3346       3346 [20] [       68:     916 ]
       458       1800       2430 [20] [       68:     916 ]
       229       1010       1514 [20] [       68:     916 ]
       121        622        926 [20] [       68:     916 ]
        65        348        708 [20] [       68:     916 ]
        37        222        528 [20] [       68:     916 ]
        20        114        420 [20] [       68:     916 ]
       456       1622       1622 [20] [       34:     456 ]
       228        850       1166 [20] [       34:     456 ]
       114        470        710 [20] [       34:     456 ]
        64        294        444 [20] [       34:     456 ]
        35        150        334 [20] [       34:     456 ]
        19         84        222 [20] [       34:     456 ]
       460       1652       1652 [20] [       34:     460 ]
       230        894       1192 [20] [       34:     460 ]
       115        474        732 [20] [       34:     460 ]
        62        306        428 [20] [       34:     460 ]
        35        166        336 [20] [       34:     460 ]
        18         92        236 [20] [       34:     460 ]
       920       3400       3400 [20] [       69:     920 ]
       460       1856       2480 [20] [       69:     920 ]
       230       1056       1560 [20] [       69:     920 ]
       126        706        950 [20] [       69:     920 ]
        68        392        740 [20] [       69:     920 ]
        37        238        558 [20] [       69:     920 ]
        20        128        432 [20] [       69:     920 ]
       460       1664       1664 [20] [       34:     460 ]
       230        882       1204 [20] [       34:     460 ]
       115        480        744 [20] [       34:     460 ]
        63        298        454 [20] [       34:     460 ]
        35        164        342 [20] [       34:     460 ]
        21         96        254 [20] [       34:     460 ]
        20         90        252 [20] [       34:     460 ]
       460       1656       1656 [20] [       34:     460 ]
       230        898       1196 [20] [       34:     460 ]
       115        502        736 [20] [       34:     460 ]
        65        310        474 [20] [       34:     460 ]
        39        184        374 [20] [       34:     460 ]
        22        104        298 [20] [       34:     460 ]
        17         88        264 [20] [       34:     460 ]
      1836       6874       6874 [20] [      137:    1836 ]
       918       3792       5038 [20] [      137:    1836 ]
       459       2146       3202 [20] [      137:    1836 ]
       248       1366       2048 [20] [      137:    1836 ]
       132        794       1568 [20] [      137:    1836 ]
        70        476       1216 [20] [      137:    1836 ]
        39        282        956 [20] [      137:    1836 ]
        20        144        730 [20] [      137:    1836 ]
       918       3400       3400 [20] [       68:     918 ]
       459       1884       2482 [20] [       68:     918 ]
       230       1026       1566 [20] [       68:     918 ]
       123        666        950 [20] [       68:     918 ]
        69        388        754 [20] [       68:     918 ]
        37        226        574 [20] [       68:     918 ]
        20        126        430 [20] [       68:     918 ]
       460       1670       1670 [20] [       34:     460 ]
       230        876       1210 [20] [       34:     460 ]
       115        480        750 [20] [       34:     460 ]
        61        310        414 [20] [       34:     460 ]
        34        174        326 [20] [       34:     460 ]
        20        108        238 [20] [       34:     460 ]
       458       1650       1650 [20] [       34:     458 ]
       229        888       1192 [20] [       34:     458 ]
       115        478        736 [20] [       34:     458 ]
        63        294        458 [20] [       34:     458 ]
        37        182        344 [20] [       34:     458 ]
        21        108        250 [20] [       34:     458 ]
        18         92        242 [20] [       34:     458 ]
       918       3394       3394 [20] [       68:     918 ]
       459       1830       2476 [20] [       68:     918 ]
       230       1058       1562 [20] [       68:     918 ]
       123        614       1040 [20] [       68:     918 ]
        69        366        792 [20] [       68:     918 ]
        40        234        604 [20] [       68:     918 ]
        22        142        456 [20] [       68:     918 ]
        17        112        436 [20] [       68:     918 ]
       458       1650       1650 [20] [       34:     458 ]
       229        888       1192 [20] [       34:     458 ]
       115        484        736 [20] [       34:     458 ]
        62        292        462 [20] [       34:     458 ]
        36        168        352 [20] [       34:     458 ]
        19         94        260 [20] [       34:     458 ]
       460       1672       1672 [20] [       34:     460 ]
       230        882       1212 [20] [       34:     460 ]
       115        512        752 [20] [       34:     460 ]
        63        336        444 [20] [       34:     460 ]
        34        166        346 [20] [       34:     460 ]
        19         98        256 [20] [       34:     460 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.016
     Coarsening: 		   0.007
            Matching: 			   0.003
            Contract: 			   0.003
     Initial Partition: 	   0.002
     Uncoarsening: 		   0.006
          Refinement: 			   0.004
          Projection: 			   0.002
     Splitting: 		   0.001
********************************************************************

 gk_mcore statistics
           coresize:        45096         nmops:         2048  cmop:      0
        num_callocs:         1198   num_hallocs:            0
       size_callocs:       857624  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:        44080   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 16
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[6.25e-02]   1=[6.25e-02]   2=[6.25e-02]   3=[6.25e-02]   4=[6.25e-02]
        5=[6.25e-02]   6=[6.25e-02]   7=[6.25e-02]   8=[6.25e-02]   9=[6.25e-02]
       10=[6.25e-02]  11=[6.25e-02]  12=[6.25e-02]  13=[6.25e-02]  14=[6.25e-02]
       15=[6.25e-02]
   Allowed maximum load imbalance: 1.001 

      3528      13384      13384 [20] [      264:    3528 ]
      1764       7372       9856 [20] [      264:    3528 ]
       882       4228       6328 [20] [      264:    3528 ]
       469       2766       3966 [20] [      264:    3528 ]
       252       1650       3112 [20] [      264:    3528 ]
       137       1032       2426 [20] [      264:    3528 ]
        74        622       1940 [20] [      264:    3528 ]
        39        322       1582 [20] [      264:    3528 ]
        21        150       1274 [20] [      264:    3528 ]
        18        126       1244 [20] [      264:    3528 ]
      1764       6598       6598 [20] [      132:    1764 ]
       882       3600       4834 [20] [      132:    1764 ]
       441       2038       3070 [20] [      132:    1764 ]
       234       1308       1888 [20] [      132:    1764 ]
       128        740       1498 [20] [      132:    1764 ]
        69        464       1134 [20] [      132:    1764 ]
        38        276        908 [20] [      132:    1764 ]
        20        140        710 [20] [      132:    1764 ]
       884       3264       3264 [20] [       66:     884 ]
       442       1762       2380 [20] [       66:     884 ]
       221        974       1496 [20] [       66:     884 ]
       118        628        894 [20] [       66:     884 ]
        62        364        680 [20] [       66:     884 ]
        33        214        510 [20] [       66:     884 ]
        18        116        398 [20] [       66:     884 ]
       440       1592       1592 [20] [       33:     440 ]
       220        848       1152 [20] [       33:     440 ]
       110        436        712 [20] [       33:     440 ]
        60        298        396 [20] [       33:     440 ]
        34        180        286 [20] [       33:     440 ]
        20        112        216 [20] [       33:     440 ]
       444       1600       1600 [20] [       33:     444 ]
       222        874       1156 [20] [       33:     444 ]
       111        484        712 [20] [       33:     444 ]
        57        278        436 [20] [       33:     444 ]
        31        152        320 [20] [       33:     444 ]
        18         88        248 [20] [       33:     444 ]
       880       3244       3244 [20] [       66:     880 ]
       440       1748       2364 [20] [       66:     880 ]
       220        974       1484 [20] [       66:     880 ]
       117        622        888 [20] [       66:     880 ]
        65        358        692 [20] [       66:     880 ]
        36        220        544 [20] [       66:     880 ]
        21        130        442 [20] [       66:     880 ]
        16         98        414 [20] [       66:     880 ]
       440       1568       1568 [20] [       33:     440 ]
       220        824       1128 [20] [       33:     440 ]
       110        448        688 [20] [       33:     440 ]
        60        286        402 [20] [       33:     440 ]
        32        152        294 [20] [       33:     440 ]
        17         88        204 [20] [       33:     440 ]
       440       1594       1594 [20] [       33:     440 ]
       220        858       1154 [20] [       33:     440 ]
       110        444        714 [20] [       33:     440 ]
        59        278        408 [20] [       33:     440 ]
        35        164        314 [20] [       33:     440 ]
        20        106        228 [20] [       33:     440 ]
      1764       6598       6598 [20] [      132:    1764 ]
       882       3552       4834 [20] [      132:    1764 ]
       441       2038       3070 [20] [      132:    1764 ]
       238       1310       1950 [20] [      132:    1764 ]
       132        780       1514 [20] [      132:    1764 ]
        73        482       1170 [20] [      132:    1764 ]
        38        278        918 [20] [      132:    1764 ]
        22        152        750 [20] [      132:    1764 ]
        17        114        726 [20] [      132:    1764 ]
       884       3258       3258 [20] [       66:     884 ]
       442       1752       2374 [20] [       66:     884 ]
       221        974       1490 [20] [       66:     884 ]
       120        666        862 [20] [       66:     884 ]
        66        386        674 [20] [       66:     884 ]
        35        224        504 [20] [       66:     884 ]
        18        122        400 [20] [       66:     884 ]
       444       1590       1590 [20] [       33:     444 ]
       222        820       1146 [20] [       33:     444 ]
       111        432        702 [20] [       33:     444 ]
        58        254        404 [20] [       33:     444 ]
        31        138        294 [20] [       33:     444 ]
        17         84        204 [20] [       33:     444 ]
       440       1566       1566 [20] [       33:     440 ]
       220        842       1126 [20] [       33:     440 ]
       110        440        686 [20] [       33:     440 ]
        60        280        388 [20] [       33:     440 ]
        34        150        294 [20] [       33:     440 ]
        18         78        206 [20] [       33:     440 ]
       880       3256       3256 [20] [       66:     880 ]
       440       1764       2376 [20] [       66:     880 ]
       220        980       1496 [20] [       66:     880 ]
       118        620        914 [20] [       66:     880 ]
        64        350        706 [20] [       66:     880 ]
        35        218        530 [20] [       66:     880 ]
        18        118        390 [20] [       66:     880 ]
       440       1558       1558 [20] [       33:     440 ]
       220        798       1118 [20] [       33:     440 ]
       110        438        678 [20] [       33:     440 ]
        60        266        404 [20] [       33:     440 ]
        35        152        300 [20] [       33:     440 ]
        21         84        226 [20] [       33:     440 ]
        18         74        212 [20] [       33:     440 ]
       440       1600       1600 [20] [       33:     440 ]
       220        852       1160 [20] [       33:     440 ]
       110        456        720 [20] [       33:     440 ]
        58        288        400 [20] [       33:     440 ]
        32        156        304 [20] [       33:     440 ]
        19         92        224 [20] [       33:     440 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.016
     Coarsening: 		   0.007
            Matching: 			   0.002
            Contract: 			   0.003
     Initial Partition: 	   0.003
     Uncoarsening: 		   0.005
          Refinement: 			   0.002
          Projection: 			   0.002
     Splitting: 		   0.000
********************************************************************

 gk_mcore statistics
           coresize:        43368         nmops:         2048  cmop:      0
        num_callocs:         1176   num_hallocs:            0
       size_callocs:       826040  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:        42352   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

15 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 0.659745 s
